### PR TITLE
torch negative frontend & test

### PR DESCRIPTION
**Pull request by ytg32**

### CODE REVIEW

**Comment by simonetgordon** [comment]
Hi,

Thanks for the contribution!

It's looking pretty good. However, before I can merge with master, merge conflicts will need to be resolved in the two files indicated.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py [hunk 1]
```diff
@@ -375,3 +375,36 @@ def test_torch_subtract(
...
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="torch",
+        fn_name="neg",
+        input=np.asarray(x, dtype=input_dtype),
+        out=None,
+    )
```
Comment (simonetgordon) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py:410]: Insert blank line at the end of the file to conform to formatting standards

**Review by simonetgordon — CHANGES_REQUESTED** [review]

**Comment by ytg32** [comment]
It seems `fn_name` argument changed to `fn_tree` in last commit, but that change was not made for `asin `function for some reason. I also corrected as `fn_tree `in `asin` function.

**Comment by kanakmi** [comment]
As someone who has recently implemented the negative function in Tensorflow frontend, I would like to point out that for this function, the valid inputs should not just be limited to floats but to all the numeric types except "uint".

**Pull request closed without merging**